lib/realtime_web/channels/realtime_channel.ex

```diff
@@ -452,14 +461,21 @@ defmodule RealtimeWeb.RealtimeChannel do
     {:reply, result, socket}
   end
 
-  def handle_in(_, _, socket) do
+  def handle_in(type, payload, socket) do
     socket = count(socket)
```

Comment:
@chasers how are we still confident in counting in this catchall given that it led to us counting incoming `nil` access tokens? we should seriously consider taking this out until we're fully confident in all the cases.

Reply:
The counter is how we found this bug in the first place :D

Plus I added that explicit Logger message there now.

Ideally we have counters everywhere and one is billable and one is non-billable but we need to be able to support more metrics for that first.